### DISCUSSION TITLE
Welcome to home-assistant-zonneplan-one Discussions!

### DISCUSSION THREAD

**Discussion by fsaris** [fsaris/home-assistant-zonneplan-one#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jdezoete87** [comment]
Hi @fsaris thank you to open this discussion place.
I got a question about the trigger, condition and action HA configuration.
I use the Zonneplan ONE integration, but what kind of configuration did i need.

The trigger is to open my electric boiler
And for the condition i use a sjabloon to use the low price sensor from Zonneplan
The action is to open my electric boiler

What did i miss or what is wrong in my flow?
